pool_automation/roles/indy_node/defaults/main.yml

```diff
@@ -5,8 +5,6 @@ indy_node_channel: master
 # TODO remove default values for versions
 indy_node_ver: 1.6.563
 indy_plenum_ver: 1.6.501
-python_indy_crypto_ver: 0.4.1
-libindy_crypto_ver: 0.4.0
```

Comment:
Do we need adding Ursa there? I'm not good at pool_automation and not sure.

Reply:
Who would have the expertise to review this? I do not.